pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>4.7.0</PackageVersion>
+    <PackageVersion>5.0.0</PackageVersion>
```

Comment:
can be deleted

Reply:
Missed these since I did a find/replace under `src`, will fixup once I'm back at my desk

Reply:
Made my comments suggestions, you can try to "commit all suggestions" and see if it works 😆